packages/wonder-blocks-dropdown/components/dropdown-core-virtualized.test.js

```diff
@@ -1,16 +1,33 @@
 //@flow
 import * as React from "react";
 import {VariableSizeList as List} from "react-window";
+// import {shallow} from "enzyme";
 import {mount} from "../../../utils/testing/mount.js";
 
 import OptionItem from "./option-item.js";
 import SeparatorItem from "./separator-item.js";
 import DropdownCoreVirtualized from "./dropdown-core-virtualized.js";
 import SearchTextInput from "./search-text-input.js";
 
-jest.useFakeTimers();
-
 describe("DropdownCoreVirtualized", () => {
+    beforeEach(() => {
+        jest.useFakeTimers();
+
+        // Jest doesn't fake out the animation frame API, so we're going to do
+        // it here and map it to timeouts, that way we can use the fake timer
+        // API to test our animation frame things.
+        jest.spyOn(global, "requestAnimationFrame").mockImplementation(
+            (fn, ...args) => setTimeout(fn, 0),
```

Comment:
Is the `...args` necessary? It seems like you could just remove it - same for the mock below, too.